xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampRequest.xml

```diff
@@ -121,14 +121,14 @@
         <Parameter Name="extensions" Type="System.Security.Cryptography.X509Certificates.X509ExtensionCollection" />
       </Parameters>
       <Docs>
-        <param name="hash">To be added.</param>
-        <param name="hashAlgorithmId">To be added.</param>
-        <param name="requestedPolicyId">To be added.</param>
-        <param name="nonce">To be added.</param>
-        <param name="requestSignerCertificates">To be added.</param>
-        <param name="extensions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hash">The pre-computed hash value to be timestamped.</param>
+        <param name="hashAlgorithmId">The Object Identifier (OID) for the hash algorithm which produced <paramref name="hash" />.</param>
+        <param name="requestedPolicyId">The Object Identifier (OID) for a timestamp policy the Timestamp Authority (TSA) should use, or <see langword="null" /> to express no preference.</param>
+        <param name="nonce">An optional nonce (number used once) to uniquely identify this request to pair it with the response. The value is interpreted as an unsigned big-endian integer and may be normalized to the encoding format.</param>
+        <param name="requestSignerCertificates">Indicates whether the Timestamp Authority (TSA) must (<see langword="true" />) or must not (<see langword="false" />) include the signing certificate in the issued timestamp token.</param>
```

Comment:
```suggestion
        <param name="requestSignerCertificates"><see langword="true" /> to indicate the Timestamp Authority (TSA) must include the signing certificate in the issued timestamp token; otherwise, <see langword="false" />.</param>
```

Reply:
Yeah I like this suggestion, the original wording was overengineered :P